docs/source/tutorial/first.rst

```diff
@@ -7,13 +7,13 @@ First Optimization
 Quadratic Function Example
 --------------------------
 
-Let us try very simple optimization in IPython shell.
+Usually, Optuna is used as a hyper-parameter optimizer, but as an example, let us try a direct optimization in IPython shell.
```

Comment:
Not sure the reader would clearly understand how `direct optimization` would differ from `hyper-parameter optimization`. How about `.. let us directly optimize a quadratic function to demonstrate its concept`, and delete the sentence below. The latter part also motivates why.

Reply:
Updated.